platform/jewel/foundation/src/main/kotlin/org/jetbrains/jewel/foundation/lazy/tree/BasicLazyTree.kt

```diff
@@ -184,7 +184,6 @@ public fun <T> BasicLazyTree(
                         }
                         .clickable(interactionSource = remember { MutableInteractionSource() }, indication = null) {
                             val item = flattenedTree[index] as Tree.Element<T>
-                            JewelLogger.getInstance("BasicLazyTree").warn("Clicky clicky: ${item.data}")
```

Comment:
I apologize for this OT change, but I saw the log and I couldn't in good conscience leave it in 🙏🏻

Reply:
Yeah, definitely needed to be removed.